src/DoctrineBatchUtils/BatchProcessing/SelectBatchIteratorAggregate.php

```diff
@@ -24,13 +24,9 @@
 final class SelectBatchIteratorAggregate implements IteratorAggregate
 {
     /** @var Traversable<mixed> */
-    private $resultSet;
-
-    /** @var EntityManagerInterface */
-    private $entityManager;
-
-    /** @var int */
-    private $batchSize;
+    private Traversable $resultSet;
+    private EntityManagerInterface $entityManager;
+    private int $batchSize;
```

Comment:
Could we add `/** @psalm-var positive-int */` to this? If not, a new issue also suffices for this purpose

Reply:
Could we add psalm specific tag in another PR?

Reply:
Most certainly: an issue also suffices - don't want to push more work on your end, if you don't feel like doing it :)